projects/commudle-admin/src/app/components/home/communities/home-community-card/home-community-card.component.scss

```diff
@@ -1,14 +1,17 @@
+@import "../../../../../assets/styles/variables.scss";
```

Comment:
Let's import it from the root path: @import '/projects/commudle-admin/src/assets/styles/variables.scss';